cli/flox-rust-sdk/src/flox.rs

```diff
@@ -253,58 +297,29 @@ impl Floxhub {
     }
 }
 
-/// Requires login with auth0 with "openid" and "profile" scopes
-/// https://auth0.com/docs/scopes/current/oidc-scopes
-/// See also: `authenticate` in `flox/src/commands/auth.rs` where we set the scopes
-#[derive(Debug, Serialize, Deserialize)]
-struct Auth0User {
-    /// full name of the user
-    name: String,
-    /// nickname of the user (e.g. github username)
-    nickname: String,
-}
-
-pub struct Auth0Client {
-    base_url: String,
-    oauth_token: String,
-}
-
-impl Auth0Client {
-    pub fn new(base_url: String, oauth_token: String) -> Self {
-        Auth0Client {
-            base_url,
-            oauth_token,
-        }
-    }
-
-    pub async fn get_username(&self) -> Result<String, reqwest::Error> {
-        let url = format!("{}/userinfo", self.base_url);
-        let client = reqwest::Client::new();
-        let request = client
-            .get(url)
-            .header(USER_AGENT, "flox cli")
-            .bearer_auth(&self.oauth_token);
-
-        let response = request.send().await?;
-
-        if response.status().is_success() {
-            let user: Auth0User = response.json().await?;
-            Ok(user.nickname)
-        } else {
-            Err(response.error_for_status().unwrap_err())
-        }
-    }
-}
-
 #[cfg(test)]
 pub mod tests {
     use std::str::FromStr;
 
-    use reqwest::header::AUTHORIZATION;
     use tempfile::TempDir;
 
     use super::*;
 
+    /// A fake floxhub token
+    ///
```

Comment:
I don't understand this comment

Reply:
A JWT consists of three parts:

1) a json header, base64 encoded
2) a json payload, base64 encoded
3) a signature (== `HMACSHA256(base64UrlEncode(header) + "." + base64UrlEncode(payload), "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA")`

This comment just shows the contents of the token when decoded ( you can verify that by pasting the token into https://jwt.io